### ISSUE TITLE
Matplotlib within screen session

### ISSUE THREAD

**Issue by nickp60** [nickp60/riboSeed#10]
Just wanted to post this here too:  there is an issue using riboSeed within `screen` sessions, as mentioned in the README and in [this issue](https://github.com/bioconda/bioconda-recipes/issues/6451) below.  As mentioned earlier, any help would be appreciated!

To reproduce the issue, try unsetting $DISPLAY and then starting a `screen` session.  Or better yet, fork the repo, and use bioconda's `simulate_travis.py` scheme to reproduce the issue.

**Comment by bgruening** [comment]
This is still a problem right?

**Comment by nickp60** [comment]
Yes, unfortunately.  I have tried all the solutions listed on the boards (moving to the non-pyplot API, adding the mpl.use('agg') to each of the scripts that plot, and changing the import order as your comment had suggested.  Based on the problem and other's issues, I am wondering if some dependancy is calling matplotlib on import, preventing me from actually setting the backend properly.  I should probably give it another go now...
